.github/workflows/upstream.yml

```
@@ -22,8 +22,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-24.04, macos-15, windows-2025]
-    uses: munich-quantum-toolkit/workflows/.github/workflows/reusable-qiskit-upstream.yml@v1.12
+        os: [ubuntu-24.04, macos-14, windows-2025]
```

```suggestion
        os: [ubuntu-24.04, macos-14, windows-2022]
```
might as well run this on the default platform we use elsewhere.